cypress/e2e/homeWork/homework/homework11.cy.js

```diff
@@ -0,0 +1,10 @@
+///<reference types="cypress" />
+
+
+context('Check the AIN website', () => {
+
+    it('Open the main page and click startups', () => {
+      cy.visit('https://ain.ua/')
+        cy.task('saveURL', 'https://ain.ua/')
```

Comment:
should you code work with cy.url()?

Reply:
yeap, added new commit, check it out, please

Reply:
you can use method cy.url() iniside your code
like this `cy.task('saveURL', cy.url())`